pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/EventDao.sql.stg

```diff
@@ -1,5 +1,85 @@
+     ett.event_trigger_type_code event_trigger_type,
+     eat.event_action_type_code event_action_type,
+     ec.post_delay_seconds,
+     ec.dispatch_to_housekeeping,
+     (SELECT
+         e.expression_text from expression e
+     WHERE e.expression_id = ec.precondition_expression_id) as precondition_expression,
+     (SELECT
+         e.expression_text from expression e
+     WHERE e.expression_id = ec.cancel_expression_id) as cancel_expression,
+     ec.max_occurrences_per_user,
+     md.gcp_topic,
+     aist.activity_instance_status_type_code as instance_status_type,
+     ast.study_activity_id as activity_status_trigger_study_activity_id,
+     wst.workflow_state_id,
+     wst.trigger_automatically,
+     dnet.dsm_notification_event_type_code as dsm_notification_event_type,
+     ann_act.message_template_id as message_template_id,
+     nt.notification_type_code as notification_type,
+     ns.service_code as notification_service_type,
+     unea.notification_template_id,
+     unea.linked_activity_id,
+     pea.pdf_document_configuration_id,
+     unpdf.generate_if_missing,
+     aica.study_activity_id as activity_instance_creation_study_activity_id,
+     cp_target.copy_target as copy_answer_target,
+     qsc.stable_id as copy_source_question_stable_id
+FROM
+     event_configuration ec
+JOIN event_trigger et ON ec.event_trigger_id = et.event_trigger_id
+JOIN event_trigger_type ett ON et.event_trigger_type_id = ett.event_trigger_type_id
+JOIN event_action ea ON ec.event_action_id = ea.event_action_id
+JOIN event_action_type eat ON ea.event_action_type_id = eat.event_action_type_id
+JOIN umbrella_study study on ec.umbrella_study_id = study.umbrella_study_id
+LEFT JOIN message_destination md on ea.message_destination_id = md.message_destination_id
+LEFT JOIN activity_status_trigger ast ON ast.activity_status_trigger_id = ec.event_trigger_id
+LEFT JOIN activity_instance_status_type aist ON ast.activity_instance_status_type_id = aist.activity_instance_status_type_id
+LEFT JOIN activity_instance ai ON ai.study_activity_id = ast.study_activity_id
+LEFT JOIN workflow_state_trigger wst on wst.event_trigger_id = et.event_trigger_id
+LEFT JOIN dsm_notification_trigger dnt ON dnt.dsm_notification_trigger_id = et.event_trigger_type_id
+LEFT JOIN dsm_notification_event_type dnet ON dnet.dsm_notification_event_type_id = dnt.dsm_notification_event_type_id
+LEFT JOIN user_announcement_event_action as ann_act on ann_act.event_action_id = ea.event_action_id
+LEFT JOIN user_notification_event_action unea on unea.user_notification_event_action_id = ea.event_action_id
+LEFT JOIN notification_type nt on nt.notification_type_id = unea.notification_type_id
+LEFT JOIN notification_service ns on ns.notification_service_id = unea.notification_service_id
+LEFT JOIN pdf_generation_event_action as pea on pea.event_action_id = ea.event_action_id
+LEFT JOIN user_notification_pdf unpdf on unpdf.user_notification_event_action_id = ea.event_action_id
```

Comment:
Each email event configuration might have one or more pdf attachments. This will result in the same issue as with `left join activity_instance`.